_component_design/chips.md

```diff
@@ -17,27 +17,87 @@ helix-ui-javascript: false
 
 <section class="static-section" markdown="1">
 
-## Introduction
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
```

Comment:
Needs update to size 8 per discussion in Slack today (19 Apr 2018)

Will need update if #493 gets merged first.